nncf/torch/accuracy_aware_training/runner.py

```diff
@@ -133,9 +135,7 @@ def dump_statistics(self, model, compression_controller):
                     self.add_tensorboard_scalar('compression/statistics/{0}'.format(key),
                                                 value, self.cumulative_epoch_count)
 
-
-    def dump_checkpoint(self, model, compression_controller):
-        checkpoint_path = osp.join(self._checkpoint_save_dir, 'acc_aware_checkpoint_last.pth')
+    def _collect_checkpoint(self, model, compression_controller):
```

Comment:
I think this method is unnecessary.

Reply:
I just added to improve code's readability